lib/common/strings.c

```diff
@@ -817,7 +818,7 @@ pcmk__parse_ll_range(const char *srcstring, long long *start, long long *end)
 
     crm_trace("Attempting to decode: [%s]", srcstring);
     if (pcmk__str_empty(srcstring) || !strcmp(srcstring, "-")) {
-        return pcmk_rc_unknown_format;
+        return ENODATA;
```

Comment:
While we're here ... if (pcmk__str_eq(srcstring, "-", pcmk__str_null_matches))

Reply:
Fixed.

Reply:
Hmm, looking at the test results maybe we should check separately so empty -> ENODATA and "-" -> bad_input

Reply:
Yeah, agreed.  Pushed another update.